tests/src/Kernel/RenderingTest.php

```diff
@@ -101,14 +101,6 @@ public function submitForm(array &$form, FormStateInterface $form_state) {}
    * @dataProvider renderingDataProvider
    */
   public function testRendering(array $structure, array $assertions): void {
-    // @todo Remove when support for 10.1.x is dropped.
-    if (!empty($structure['core_version'])) {
-      if (!$this->shouldBeTested($structure['core_version'])) {
```

Comment:
We need to remove the method called here.

Reply:
Removed